tezos-sig-gating-contracts/scripts/manual/mintOffchain.ts

```diff
@@ -76,27 +76,28 @@ async function main() {
   // const senderAddress = "tz1TiFzFCcwjv4pyYGTrnncqgq17p59CzAE2";
 
   const signerBob = new InMemorySigner(
-    "edskS7YYeT85SiRZEHPFjDpCAzCuUaMwYFi39cWPfguovTuNqxU3U9hXo7LocuJmr7hxkesUFkmDJh26ubQGehwXY8YiGXYCvU"
+    "edsk3QoqBuvdamxouPhin7swCvkQNgq4jP5KZPbwWNnwdZpSpJiEbq"
+    // "edskS7YYeT85SiRZEHPFjDpCAzCuUaMwYFi39cWPfguovTuNqxU3U9hXo7LocuJmr7hxkesUFkmDJh26ubQGehwXY8YiGXYCvU"
   ); // bob private key
   // const signerAddress = "tz1TiFzFCcwjv4pyYGTrnncqgq17p59CzAE2";
 
   try {
     console.log("Attempt mint #4 in ghostnet");
     // INPUTS
-    const functioncall_contract = "KT1C3T9RuGHTyj9bPJxHhtzq7ZqtA7J2pKEb"; //"KT1AoU1mrLRSM2zouUVkvLz2UHo1on4UAFBF";
+    const functioncall_contract = "KT1KNhkV7c7bCdFvCXM6LNxbgzSnRxEoXP8Y"; //"KT1AoU1mrLRSM2zouUVkvLz2UHo1on4UAFBF";
     const functioncall_name = "%mint_gated";
     const functioncall_params = {
-      owner: "tz2SrmyZjTj8Z1SxU4X2rp2PgadreRLtLHMC",
-      token_id: "4",
+      owner: "tz1fon1Hp3eRff17X82Y3Hc2xyokz33MavFF",
```

Comment:
and this

Reply:
this address must be specified everytime we want to use this script .. .as you can see (it was a "tz2...") which is a generated address by our test . so I don't think it is relevant here